templates/kubernetes/overlays/devenv/deployment.yml

```diff
@@ -0,0 +1,17 @@
+apiVersion: apps/v1
```

Comment:
Can we just call this "dev" instead of "devenv"?

Reply:
good